MovieChecker/MovieChecker/MovieList.h

```diff
@@ -0,0 +1,24 @@
+//
+//  MovieList.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/27/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol MovieListDelegate <NSObject>
+
+- (void)synchMoviesWithArray:(NSArray *)movies;
+
+@end
+
+@interface MovieList : NSObject
+
+@property (nonatomic, strong) NSArray *movies;
+@property (nonatomic, weak) id <MovieListDelegate> delegate;
+
+- (id)initWithDelegate:(id)delegate;
```

Comment:
It's a bit awkward that the action is caused by the initializer... it might be better for the action to be a separate method so that when this is consumed, it's clear than an action is undertaken and so that it'll be possible to repeat the action without having to throw away the object and get a new one.